R/zzz.R

```diff
@@ -1,5 +1,12 @@
 .onLoad <- function(libname, pkgname) {
 
+  # if the option set before load, keep it, else look for an env var
+  # if it isn't one of the false-y settings, then will set to true
+  if (is.null(getOption("rbabylon.DEV_no_min_version")) &&
+      !Sys.getenv("RBABYLON_DEV_NO_MIN_VERSION", unset = "FALSE") %in% c("0", "FALSE", "false")) {
```

Comment:
Why do you want to just check the env var on load and set the option? Why not just check the env var in the function? 

Doing this with an environment variable makes sense to me, but I don't understand why we bring options into it. If we do it this way then you could set the env var to TRUE but then it doesn't take effect until you reload the package. Why is that desirable?

Reply:
This way you can more easily manage it in CI only, by setting an environment variable, but can manage it interactively via options to change the behavior at any point in time. This keeps us still providing the same "interface" to users (set options), makes the option extra visible as a dev option, but also gives us a clean CI/global/system implementation via env vars.

This is actually the pattern that I'd like to move to for basically all options/pkg settings

Reply:
Per discussion offline, I think if we're going to implement a system of configuration inheritance like this we should do it in a package so that it remains consistent. Adding this complexity already worries me, but adding it without enforcing consistency _really_ worries me.

I think we can go ahead and merge this and we'll link to it as we develop the new package so that we replace this code with the new package code once it exists. 

I'm not too worried about any breaking changes (to the `no_min_version` API) that could potentially happen, especially since this is only intended for developers.